suite-native/atoms/src/VerticalSeparator.tsx

```diff
@@ -0,0 +1,41 @@
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Translation, TxKeyPath } from '@suite-native/intl';
+
```

Comment:
Nice. I have checked figma and it looks like we will have more of this textual dividers. Once merged, I'll reuse it in device switcher